pkg/costmodel/costmodel.go

```diff
@@ -2437,6 +2442,9 @@ func (cm *CostModel) QueryAllocation(window kubecost.Window, resolution, step ti
 		IdleByNode:                            idleByNode,
 		IncludeAggregatedMetadata:             includeAggregatedMetadata,
 	}
+	if namespace != "" {
+		opts.Filter = ops.Eq(afilter.FieldNamespace, namespace)
+	}
```

Comment:
This isn't quite what I was looking for. Filters can be arbitrary strings, and to match convention they should come under a query param `filter`. After that, you can use code like that which I linked earlier to compile that filter here, rather than manually making an `ops.Eq` just for namespaces. Your query would then be `/allocation/compute?filter=namespace:"kube-system"`, for example.